schema-loader/src/main/java/com/scalar/db/schemaloader/core/SchemaOperator.java

```diff
@@ -35,12 +39,34 @@ public SchemaOperator(DatabaseConfig dbConfig, boolean isStorageSpecificCommand)
   public SchemaOperator(
       DistributedStorageAdmin admin,
       ConsensusCommitAdmin consensusCommitAdmin,
-      boolean isStorageSpecificCommand) {
-    this.isStorageSpecificCommand = isStorageSpecificCommand;
+      boolean separateCoordinatorTable) {
+    this.separateCoordinatorTable = separateCoordinatorTable;
     this.admin = admin;
     this.consensusCommitAdmin = consensusCommitAdmin;
   }
 
+  public void createTables(Path schemaFile, Map<String, String> metaOptions)
+      throws SchemaOperatorException {
+    List<Table> tableList;
+    try {
+      tableList = SchemaParser.parse(schemaFile, metaOptions);
+    } catch (SchemaException | IOException e) {
+      throw new SchemaOperatorException("Parsing the schema file failed.", e);
+    }
+    createTables(tableList, metaOptions);
+  }
+
+  public void createTables(String serializedSchemaJson, Map<String, String> metaOptions)
+      throws SchemaOperatorException {
+    List<Table> tableList;
+    try {
+      tableList = SchemaParser.parse(serializedSchemaJson, metaOptions);
+    } catch (SchemaException e) {
+      throw new SchemaOperatorException("Parsing the serialized schema json failed.", e);
+    }
+    createTables(tableList, metaOptions);
+  }
+
   public void createTables(List<Table> tableList, Map<String, String> metaOptions)
```

Comment:
```suggestion
  private void createTables(List<Table> tableList, Map<String, String> metaOptions)
```

Reply:
We have the tests for this method so I set its type to package-private